Error on install since yesterday for cloudflare langchain

### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

not relevant

### Error Message and Stack Trace (if applicable)

```
0 verbose cli /usr/bin/node /usr/bin/npm
1 info using npm@10.8.3
2 info using node@v22.8.0
3 silly config load:file:/usr/lib/node_modules/npm/npmrc
4 silly config load:file:/home/noah/test/.npmrc
5 silly config load:file:/home/noah/.npmrc
6 silly config load:file:/etc/npmrc
7 verbose title npm install @langchain/cloudflare @langchain/core
8 verbose argv "install" "@langchain/cloudflare" "@langchain/core"
9 verbose logfile logs-max:10 dir:/home/noah/.npm/_logs/2024-09-14T09_23_35_608Z-
10 verbose logfile /home/noah/.npm/_logs/2024-09-14T09_23_35_608Z-debug-0.log
11 silly packumentCache heap:4345298944 maxSize:1086324736 maxEntrySize:543162368
12 silly logfile start cleaning logs, removing 1 files
13 silly logfile done cleaning log files
14 silly idealTree buildDeps
15 silly fetch manifest @langchain/cloudflare@*
16 silly packumentCache full:https://registry.npmjs.org/@langchain%2fcloudflare cache-miss
17 http fetch GET 200 https://registry.npmjs.org/@langchain%2fcloudflare 8ms (cache hit)
18 silly packumentCache full:https://registry.npmjs.org/@langchain%2fcloudflare set size:46349 disposed:false
19 silly fetch manifest @langchain/core@*
20 silly packumentCache full:https://registry.npmjs.org/@langchain%2fcore cache-miss
21 http fetch GET 200 https://registry.npmjs.org/@langchain%2fcore 5ms (cache hit)
22 silly packumentCache full:https://registry.npmjs.org/@langchain%2fcore set size:1861746 disposed:false
23 silly placeDep ROOT @langchain/cloudflare@0.1.0 OK for: test@1.0.0 want: *
[...]
835 silly fetch manifest couchbase@*
836 silly packumentCache full:https://registry.npmjs.org/couchbase cache-hit
837 silly fetch manifest d3-dsv@*
838 silly packumentCache full:https://registry.npmjs.org/d3-dsv cache-hit
839 silly fetch manifest epub2@*
840 silly packumentCache full:https://registry.npmjs.org/epub2 cache-hit
841 silly fetch manifest html-to-text@*
842 silly packumentCache full:https://registry.npmjs.org/html-to-text cache-hit
843 silly fetch manifest ignore@*
844 silly packumentCache full:https://registry.npmjs.org/ignore cache-hit
845 silly fetch manifest ioredis@*
846 silly packumentCache full:https://registry.npmjs.org/ioredis cache-miss
847 http fetch GET 200 https://registry.npmjs.org/ioredis 5ms (cache hit)
848 silly packumentCache full:https://registry.npmjs.org/ioredis set size:2606494 disposed:false
849 silly fetch manifest mammoth@*
850 silly packumentCache full:https://registry.npmjs.org/mammoth cache-hit
851 silly fetch manifest mongodb@*
852 silly packumentCache full:https://registry.npmjs.org/mongodb cache-miss
853 http fetch GET 200 https://registry.npmjs.org/mongodb 4ms (cache hit)
854 silly packumentCache full:https://registry.npmjs.org/mongodb set size:2230562 disposed:false
855 silly fetch manifest notion-to-md@*
856 silly packumentCache full:https://registry.npmjs.org/notion-to-md cache-hit
857 silly fetch manifest officeparser@*
858 silly packumentCache full:https://registry.npmjs.org/officeparser cache-hit
859 silly fetch manifest pdf-parse@*
860 silly packumentCache full:https://registry.npmjs.org/pdf-parse cache-hit
861 silly fetch manifest playwright@*
862 silly packumentCache full:https://registry.npmjs.org/playwright cache-miss
863 http fetch GET 200 https://registry.npmjs.org/playwright 19ms (cache hit)
864 silly packumentCache full:https://registry.npmjs.org/playwright set size:13151855 disposed:false
865 silly fetch manifest sonix-speech-recognition@*
866 silly packumentCache full:https://registry.npmjs.org/sonix-speech-recognition cache-hit
867 silly fetch manifest srt-parser-2@*
868 silly packumentCache full:https://registry.npmjs.org/srt-parser-2 cache-hit
869 silly fetch manifest typeorm@*
870 silly packumentCache full:https://registry.npmjs.org/typeorm cache-miss
871 http fetch GET 200 https://registry.npmjs.org/typeorm 40ms (cache hit)
872 silly packumentCache full:https://registry.npmjs.org/typeorm set size:23854556 disposed:false
873 silly fetch manifest mysql2@^2.2.5 || ^3.0.1
874 silly packumentCache full:https://registry.npmjs.org/mysql2 cache-hit
875 silly fetch manifest pg@^8.5.1
876 silly packumentCache full:https://registry.npmjs.org/pg cache-hit
877 silly fetch manifest web-auth-library@*
878 silly packumentCache full:https://registry.npmjs.org/web-auth-library cache-hit
879 silly fetch manifest ws@*
880 silly packumentCache full:https://registry.npmjs.org/ws cache-hit
881 silly fetch manifest youtube-transcript@*
882 silly packumentCache full:https://registry.npmjs.org/youtube-transcript cache-hit
883 silly fetch manifest youtubei.js@*
884 silly packumentCache full:https://registry.npmjs.org/youtubei.js cache-hit
885 verbose stack TypeError: Cannot read properties of null (reading 'edgesOut')
885 verbose stack     at #loadPeerSet (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1286:38)
885 verbose stack     at async #loadPeerSet (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1294:11)
885 verbose stack     at async #buildDepStep (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:901:11)
885 verbose stack     at async Arborist.buildIdealTree (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:181:7)
885 verbose stack     at async Promise.all (index 1)
885 verbose stack     at async Arborist.reify (/usr/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/reify.js:131:5)
885 verbose stack     at async Install.exec (/usr/lib/node_modules/npm/lib/commands/install.js:150:5)
885 verbose stack     at async Npm.exec (/usr/lib/node_modules/npm/lib/npm.js:207:9)
885 verbose stack     at async module.exports (/usr/lib/node_modules/npm/lib/cli/entry.js:74:5)
886 error Cannot read properties of null (reading 'edgesOut')
887 silly unfinished npm timer reify 1726305815743
888 silly unfinished npm timer reify:loadTrees 1726305815745
889 silly unfinished npm timer idealTree:buildDeps 1726305815754
890 silly unfinished npm timer idealTree:node_modules/@langchain/core 1726305815892
891 verbose cwd /home/noah/test
892 verbose os Linux 6.10.9-arch1-2
893 verbose node v22.8.0
894 verbose npm  v10.8.3
895 verbose exit 1
896 verbose code 1
897 error A complete log of this run can be found in: /home/noah/.npm/_logs/2024-09-14T09_23_35_608Z-debug-0.log

```

### Description

Hello, since yesterday, `@langchain/cloudflare` generate error on install `npm ERR! Cannot read properties of null (reading 'edgesOut')`. With 0.0.7 and 0.1.0. This occurs on my system but also on cloudflare page, thus I can't publish any update to my website

### System Info

"
langchain@0.3.0 | MIT | deps: 12 | versions: 293
Typescript bindings for langchain
https://github.com/langchain-ai/langchainjs/tree/main/langchain/

keywords: llm, ai, gpt3, chain, prompt, prompt engineering, chatgpt, machine learning, ml, openai, embeddings, vectorstores

dist
.tarball: https://registry.npmjs.org/langchain/-/langchain-0.3.0.tgz
.shasum: 006db05e5c774db83725194336c01e01bcace703
.integrity: sha512-gvB8ikvCxL6KWS0HN89gdHMDNi9gUKz3MRMU6/TqC86T0v9Bvg2mIS3pF1qBaoC3/SXG404+uUduOiX7BOXmJQ==
.unpackedSize: 2.9 MB

dependencies:
@langchain/openai: >=0.1.0 <0.4.0        openapi-types: ^12.1.3                   
@langchain/textsplitters: >=0.0.0 <0.2.0 p-retry: 4                               
js-tiktoken: ^1.0.12                     uuid: ^10.0.0                            
js-yaml: ^4.1.0                          yaml: ^2.2.1                             
jsonpointer: ^5.0.1                      zod-to-json-schema: ^3.22.3              
langsmith: ~0.1.40                       zod: ^3.22.4                             

maintainers:
- nfcampos <nuno@boringbits.io>
- jacoblee93 <jacoblee93@gmail.com>
- andrewnguonly <andrewnguonly@gmail.com>
- davidduong <david@duong.cz>
- hwchase17 <hw.chase.17@gmail.com>
- basproul <braceasproul@gmail.com>

dist-tags:
latest: 0.3.0     next: 0.3.0-rc.0  

published 11 hours ago by jacoblee93 <jacoblee93@gmail.com>"
linux
node : v22.8.0
npm : 10.8.3

--- a/langchain-core/package.json
+++ b/langchain-core/package.json
@@ -37,7 +37,7 @@
     "camelcase": "6",
     "decamelize": "1.2.0",
     "js-tiktoken": "^1.0.12",
-    "langsmith": "^0.1.43",
+    "langsmith": "^0.1.56-rc.1",
     "mustache": "^4.2.0",
     "p-queue": "^6.6.2",
     "p-retry": "4",

--- a/langchain/package.json
+++ b/langchain/package.json
@@ -524,7 +524,7 @@
     "js-tiktoken": "^1.0.12",
     "js-yaml": "^4.1.0",
     "jsonpointer": "^5.0.1",
-    "langsmith": "~0.1.40",
+    "langsmith": "^0.1.56-rc.1",
     "openapi-types": "^12.1.3",
     "p-retry": "4",
     "uuid": "^10.0.0",

